.github/workflows/get-started-tests.yml

```diff
@@ -17,14 +17,15 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest]
+        go-version: ['1.22', '1.23']
     runs-on: ${{ matrix.os }}
     steps:
     - name: Checkout code
       uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
     - name: Install Go
       uses: actions/setup-go@0a12ed9d6a96ab950c8f026ed9f722fe0da7ef32
       with:
-        go-version: '1.22'
+        go-version: ${{ matrix.go-version }}
```

Comment:
Amending the rule of thumb above: two is good here because it's again about the version of Go a user may have.